Defined "food desert" (#353)
tatianamac/selfdefined: 11ty/definitions/food-desert.md

---

> title: Food Desert

Reviewer: ```suggestion
title: food desert
```

slug: food-desert
defined: true
speech: noun

> excerpt: an area that has limited access to affordable and nutritious food

Reviewer: ```suggestion
excerpt: an area that has limited access to affordable and nutritious food
reading:
  - text: "Food Empowerment Project: Food Deserts"
    href: https://foodispower.org/access-health/food-deserts/
```

---

> A geographic area where residents’ access to affordable, healthy food options (especially fresh fruits and vegetables) is restricted or nonexistent due to the absence of grocery stores within convenient traveling distance.

Reviewer: ```suggestion
a geographic area where residents’ access to affordable, healthy food options (especially fresh fruits and vegetables) is restricted or non-existent due to the absence of grocery stores within convenient traveling distance.
```
Would including "farmers' markets" here make sense?

> The other defining characteristic of food deserts is socio-economic: that is, they are most commonly found in black and brown communities and low-income areas (where many people don’t have cars). Studies have found that wealthy districts have three times as many supermarkets as poor ones do, [2] that white neighborhoods contain an average of four times as many supermarkets as predominantly black ones do, and that grocery stores in African-American communities are usually smaller with less selection.

Reviewer: ```suggestion
According to [Food Empowerment](https://foodispower.org/access-health/food-deserts/), "the other defining characteristic of food deserts is socio-economic: that is, they are most commonly found in black and brown communities and low-income areas (where many people don’t have cars). Studies have found that wealthy districts have three times as many supermarkets as poor ones do, that white neighborhoods contain an average of four times as many supermarkets as predominantly black ones do, and that grocery stores in African-American communities are usually smaller with less selection."
```

> Find more information at the [Food Empowerment Project](https://foodispower.org/access-health/food-deserts/)

Reviewer: ```suggestion

```
Added this link in additional reading above.
